package.json

```diff
@@ -51,18 +51,17 @@
     "@total-typescript/ts-reset": "^0.3.7",
     "@types/eslint": "^8.37.0",
     "@types/node": "^18.16.0",
-    "@types/prettier": "^2.7.2",
     "@typescript-eslint/eslint-plugin": "6.0.0",
     "@typescript-eslint/parser": "6.0.0",
     "eslint": "^8.40.0",
     "eslint-config-prettier": "^8.8.0",
```

Comment:
i believe there is a v9 available for this, not sure if it's related to prettier v3 though

Reply:
yup, your right, bumped that